boards/opentitan/src/main.rs

```diff
@@ -213,7 +213,7 @@ pub unsafe fn reset_handler() {
 
     let lldb = components::lldb::LowLevelDebugComponent::new(board_kernel, uart_mux).finalize(());
 
-    aes_test::run_aes128_ecb();
+    //aes_test::run_aes128_ecb();
```

Comment:
Maybe just remove the line instead of commenting out code.

Reply:
Was matching what other boards have done to indicate tests e.g. https://github.com/tock/tock/blob/master/boards/imix/src/main.rs#L500

Reply:
It seems like a bad idea to have commented out code, it's just prone to bitrotting. As it's only 1 line I still think it should be removed.